### DISCUSSION TITLE
[QUESTION] Is Figma Tokens meant to be used for component tokens?

### DISCUSSION THREAD

**Discussion by aranoe** [six7/figma-tokens#224] [Q&A]
If yes: How should you do this? 
If not: Is it planned to support component tokens?
 
 Here is an example hierarchy:

`global` -> ( `alias` -> `...` -> `alias`) -> `component`
 
Concrete example: 

 `color.error` -> `input.icon.error.color`
 `color.primary` -> `form.interaction.color` -> `button.primary.background-color`
 `color.primary.light` -> `form.interaction.hover.color` -> `button.primary.hover.background-color`
 
 
 Right now i only see it managable for `global` and `alias` tokens.

**Comment by six7** [comment]
Sure, you can do that. You can just create these tokens that reference on another.

The UX isn't the best for that right now, you could use the searchbar to filter for tokens that you want to use right now, but it could be improved.
